public/js/gulliver.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-env browser */
+
+/**
+ * Note: window.gapiReady must be a promise that resolves to
+ * window.gapi.
+ *
+ * @template T
+ * @param {string} name the library to load
+ * @return {Promise<T>}
+ */
+function gapiLoad(name) {
+  return new Promise((resolve, reject) => {
```

Comment:
ES6 Promises and arrow notation here, and elsewhere. For this code to run everywhere, we can either write ES5 by hand, or add a transpiler. If a transpiler (which I'd prefer), need to add this to the build process. (And also have a build process…)